Monika After Story/game/script-topics.rpy

```diff
@@ -15532,6 +15532,114 @@ label monika_being_herself:
     extend 5eua "Well, {i}Just Me.{/i}"
     return
 
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_being_deleted",
+            prompt="Are you still mad at me for deleting you ?",
+            category=["ddlc", "monika"],
+            pool=True,
+            action=EV_ACT_POOL,
```

Comment:
These two contradict each other.

`EV_ACT_POOL` means once the conditional passes, the event will be `pool`ed, however you're starting it as pooled.